### ISSUE TITLE
Toast stops working on AOT.

### ISSUE THREAD

**Issue by kodeine** [driftyco/ionic-app-scripts#947]
after aot, toast component stops displaying toasts. Without aot, it works fine.

```
Cordova CLI: 6.3.1 
Ionic Framework Version: 3.1.1
Ionic CLI Version: 2.2.3
Ionic App Lib Version: 2.2.1
Ionic App Scripts Version: 1.3.6
ios-deploy version: Not installed
ios-sim version: Not installed
OS: Linux 4.2
Node Version: v6.6.0
Xcode version: Not installed
```

**Comment by danbucholtz** [comment]
@kodeine,

How can I recreate this issue?

Thanks,
Dan

**Comment by danbucholtz** [comment]
This should be fixed in 1.3.7.

Thanks,
Dan
